Lightyear/Core.idr

```diff
@@ -153,4 +153,13 @@ satisfy : (Monad m, Stream tok str)
                    => (tok -> Bool)
                    -> ParserT str m tok
 satisfy p = satisfyMaybe (\t => if p t then Just t else Nothing)
+
+||| Succeed if and only if the argument parser fails.
+notFollowedBy : ParserT str m tok -> ParserT str m ()
```

Comment:
Well, I find it strange too, I just went with it because that’s what Parsec (and IIRC the Parsers package) call it. Nor have I found a name that seems truly *right*. I’d go with whatever you like best.